scalr/analysis/dge_lmem.py

```diff
@@ -253,12 +254,14 @@ def plot_lmem_dge_result(self,
 
                 lmem_res_df[f'-log10_{pval_col}'] = -np.log10(
                     lmem_res_df[pval_col])
-                down_reg_genes_idx = (lmem_res_df[coef_col] < (
-                    self.coef_threshold)) & (lmem_res_df[f'-log10_{pval_col}']
-                                             >= (neg_log10_pval))
-                up_reg_genes_idx = (lmem_res_df[coef_col] > (
-                    self.coef_threshold)) & (lmem_res_df[f'-log10_{pval_col}']
-                                             >= (neg_log10_pval))
+                down_reg_genes_idx = (lmem_res_df[coef_col] <
+                                      (self.coef_threshold)) & (
+                                          lmem_res_df[f'-log10_{pval_col}'] >=
```

Comment:
These changes are introduced due to yapf only, right?

Reply:
Mostly because of yapf,  code changes - 129